tests/simpleline_tests/adv_widgets_test.py

```diff
@@ -22,7 +22,7 @@
 
 from io import StringIO
 
-from tests.simpleline_tests import UtilityMixin
+from . import UtilityMixin
```

Comment:
Have you tried to use `simpleline_tests` instead of `.`? I think it could work if you start the discovery in `/tests`.

Reply:
Yes it should probably work but this seems better solution to me. It's explicitly saying that the import is just a folder above. No PYTHONPATH mystification from the outside.

Reply:
However, `simpleline_tests` is a real package with a unique (enough) name, it not just a folder. I would avoid relative imports at all costs, but that's up to you.